Automattic/wp-calypso
# Update SVGs to use unique IDs

Related to # pe7F0s-1KM-p2

## Proposed Changes

* Updates SVGs in calypso to have unique ID names, to prevent conflicts between referencing clip paths, masks, etc. creating visual bugs as noted in the above p2.
* I have gone through calypso searching for generic id generation in svgs like `id="a"`, in those files i have ran replacements for all ids and corresponding references with a general prefix `file-name-a` or `file-name-svg-a`.

## Testing Instructions



* Go to the files changed tab and inspect the SVG before and afters shown closely. Verify there are no visual regressions.
    * Note for this I see one case where the after is different than the before, and this seems like we may have fixed a visual bug since the after looks correct and the before looks broken.

<img width="914" alt="Screenshot 2024-03-28 at 3 05 39 PM" src="https://github.com/Automattic/wp-calypso/assets/28742426/9fc22e44-9d0e-4baf-a617-b6c6aed311cd">

* For jsx files containing SVGs, load this branch of calypso to view them in action and verify no regressions. If you are unable to find the location of some of these, you can import the svg component and place it in a location you can view (inside my home, in a header on the reader, etc. it should not matter... just matters that these SVGs are still the same visual as expected.

## Pre-merge Checklist
